### PR TITLE
Appstream builder in pruner

### PR DESCRIPTION
**Pull request by praiskup**

### CODE REVIEW

File: backend/run/copr_prune_results.py [hunk 1]
```diff
@@ -170,15 +176,21 @@ def prune_project(self, project_path, username, projectdir):
         LOG.info("projectname = %s", projectname)
 
         try:
-            if uses_devel_repo(self.opts.frontend_base_url, username, projectname):
+            project_info = get_project_info(self.opts.frontend_base_url,
+                                            username, projectname)
+
+            if uses_devel_repo(self.opts.frontend_base_url, username,
+                               projectname, project_info):
```
Comment (FrostyX) [inline backend/run/copr_prune_results.py:187]: Maybe we can go as far as removing the whole `uses_devel_repo` function, and using only

```
if bool(project.get("devel_mode", False)):
    ...
```

Or I overlooked something?

**Review by FrostyX — COMMENTED** [review]
I agree, this is a big improvement.
Only one comment, otherwise LGTM

File: backend/run/copr_prune_results.py (hunk 1 above)
Comment (praiskup) [reply backend/run/copr_prune_results.py:187]: The method is used in multiple places I think.

**Comment by praiskup** [comment]
It appears pruner is entirely broken.  Thus not hot-fixing, we need to resolve #2750 first.

**Pull request merged**